### DISCUSSION TITLE
ShoppingList questions

### DISCUSSION THREAD

**Discussion by BeaglySmeagly** [TandoorRecipes/recipes#332] [Q&A]
- Is shoppinglist item ordering something that’ll be added?
- After adding an recipe to the shopping list, there could be unnecessary stuff in it like herbs, oil, spices etc. Is there a plan to make this more adjustable?

thanks for sharing such great household addition

**Comment by vabene1111** [comment]
This is the overall roadmap for shopping lists https://github.com/vabene1111/recipes/issues/114

1.  Ordering is definitely planned, also in an automated manner 
2. it is planned to have an attribute on each food item that can determine if the food is supposed to be shown in the shopping list

To implement those changes i will most likely have to rewrite a bunch of stuff since that will make some things possible in the first place so dont expect that to happen very soon.
